packages/core/test/financial-templates/expiring-multiparty/Liquidatable.js

```diff
@@ -581,16 +581,16 @@ contract("Liquidatable", function(accounts) {
 
       const startingTime = await liquidationContract.getCurrentTime();
       let expectedTimestamp = toBN(startingTime)
-        .add(liquidationLiveness)
+        .add(withdrawalLiveness)
         .toString();
 
-      assert(
+      assert.equal(
         expectedTimestamp,
         (await liquidationContract.positions(sponsor)).withdrawalRequestPassTimestamp.toString()
       );
 
       // Advance time by half of the liveness duration.
-      await liquidationContract.setCurrentTime(startingTime.add(liquidationLiveness.divn(2)).toString());
+      await liquidationContract.setCurrentTime(startingTime.add(withdrawalLiveness.divn(2)).toString());
```

Comment:
This is one part of the false positive test: in this file, `withdrawalLiveness = 3600` but non-intuitively `liquidationLiveness = 3600 * 3`. So, advancing time forward by half of the liquidation liveness actually **expires the withdrawal** which due to the contract bug allows the `withdrawalRequestPassTimestamp` to reset!

Reply:
unbelievable